api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -291,9 +291,9 @@ type AgentClusterInstallStatus struct {
 	// +optional
 	ValidationsInfo common.ValidationsStatus `json:"validationsInfo,omitempty"`
 
-	// MirrorRegistryConfigurationInfo contains the mirror registry configuration for this cluster
+	// MirrorRegistrySuccessfullyApplied represents the status of successfully applying the mirror registry config map.
 	// +optional
-	MirrorRegistryConfigurationInfo *MirrorRegistryConfigurationInfo `json:"mirrorRegistryConfigurationInfo,omitempty"`
+	MirrorRegistrySuccessfullyApplied bool `json:"MirrorRegistrySuccessfullyApplied,omitempty"`
```

Comment:
Generally boolean fields are not a good choice in APIs like this, but again if the `SpecSynced` condition is succeeding then the user can be certain that their desired state has been applied. There's no reason for additional fields to tell them that the controller is working as it should.

We don't have this kind of field for extra manifests, or ignition overrides, right?